src/utils/abi.ts

```diff
@@ -0,0 +1,14 @@
+import { Address, Chain } from 'viem';
+
+import { getContracts } from '@/deploys';
+import { seaportABI } from '@/generated/blockchain/seaport';
+import { areSameAddress } from '@/utils/string';
+
+export const getAbi = (contractAddress: Address, chain: Chain) => {
+  const contracts = getContracts(chain);
+  if (areSameAddress(contractAddress, contracts.Seaport)) {
+    return seaportABI;
+  }
+
+  return [];
```

Comment:
Looks weird to have this as a util when it's sth super specific to seaport. Would probably move to generic contract instantiation for now